packages/core/src/http-middlewares/before/prepare-request.js

```diff
@@ -52,6 +52,7 @@ const coerceBody = (req) => {
 
   // auto coerce form if header says so
   if (contentType === FORM_TYPE && req.body && !_.isString(req.body)) {
+    normalizeEmptyBodyFields(req);
```

Comment:
Similar to what is done [here](https://github.com/zapier/zapier-platform/blob/2636637b61f74c8804d95e56f2b1a841519196b8/packages/core/src/http-middlewares/before/prepare-request.js#L66).